test_bad_file_format failing with tskit v0.6.4 ?

Hi, didn't have time to investigate much but in my packaging attempts for guix I just noticed that the test `test_bad_file_format` was failing after upgrading tskit to v0.6.4 with the following error:

```
=================================== FAILURES ===================================
__________________ TestCompressSemantics.test_bad_file_format __________________
[gw1] linux -- Python 3.11.11 /gnu/store/y5vz9h983i8cg01cxz0zdywz715fxxs8-python-wrapper-3.11.11/bin/python

self = <test_cli.TestCompressSemantics testMethod=test_bad_file_format>

    def test_bad_file_format(self):
        self.assertTrue(self.trees_path.exists())
        with open(str(self.trees_path), "w") as f:
            f.write("xxx")
        with mock.patch("tszip.cli.exit", side_effect=TestException) as mocked_exit:
            with self.assertRaises(TestException):
                self.run_tszip([str(self.trees_path)])
>           mocked_exit.assert_called_once_with(
                f"Error loading '{self.trees_path}': File not in kastore format. Either"
                f" the file is corrupt or it is not a tskit tree sequence file. It may"
                f" be a legacy HDF file upgradable with `tskit upgrade` or a compressed"
                f" tree sequence file that can be decompressed with `tszip`."
            )

tests/test_cli.py:290:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/gnu/store/h6js234mh24sbgxaz5fq750pq9r8razq-python-3.11.11/lib/python3.11/unittest/mock.py:951: in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <MagicMock name='exit' id='140734010120912'>
args = ("Error loading '/tmp/guix-build-python-tszip-0.2.5.drv-0/tszip_cli_1yijv2dh/msprime.trees': File not in kastore forma...y HDF file upgradable with `tskit upgrade` or a compressed tree sequence file that can be decompressed with `tszip`.",)
kwargs = {}
expected = call("Error loading '/tmp/guix-build-python-tszip-0.2.5.drv-0/tszip_cli_1yijv2dh/msprime.trees': File not in kastore f...cy HDF file upgradable with `tskit upgrade` or a compressed tree sequence file that can be decompressed with `tszip`.")
actual = call("Error loading '/tmp/guix-build-python-tszip-0.2.5.drv-0/tszip_cli_1yijv2dh/msprime.trees': File not in kastore f...th `tskit upgrade` from tskit version<0.6.2 or a compressed tree sequence file that can be decompressed with `tszip`.")
_error_message = <function NonCallableMock.assert_called_with.<locals>._error_message at 0x7fff30af2ca0>
cause = None

    def assert_called_with(self, /, *args, **kwargs):
        """assert that the last call was made with the specified arguments.

        Raises an AssertionError if the args and keyword args passed in are
        different to the last call to the mock."""
        if self.call_args is None:
            expected = self._format_mock_call_signature(args, kwargs)
            actual = 'not called.'
            error_message = ('expected call not found.\nExpected: %s\n  Actual: %s'
                    % (expected, actual))
            raise AssertionError(error_message)

        def _error_message():
            msg = self._format_mock_failure_message(args, kwargs)
            return msg
        expected = self._call_matcher(_Call((args, kwargs), two=True))
        actual = self._call_matcher(self.call_args)
        if actual != expected:
            cause = expected if isinstance(expected, Exception) else None
>           raise AssertionError(_error_message()) from cause
E           AssertionError: expected call not found.
E           Expected: exit("Error loading '/tmp/guix-build-python-tszip-0.2.5.drv-0/tszip_cli_1yijv2dh/msprime.trees': File not in kastore format. Either the file is corrupt or it is not a tskit tree sequence file. It may be a legacy HDF file upgradable with `tskit upgrade` or a compressed tree sequence file that can be decompressed with `tszip`.")
E             Actual: exit("Error loading '/tmp/guix-build-python-tszip-0.2.5.drv-0/tszip_cli_1yijv2dh/msprime.trees': File not in kastore format. Either the file is corrupt or it is not a tskit tree sequence file. It may be a legacy HDF file upgradable with `tskit upgrade` from tskit version<0.6.2 or a compressed tree sequence file that can be decompressed with `tszip`.")

/gnu/store/h6js234mh24sbgxaz5fq750pq9r8razq-python-3.11.11/lib/python3.11/unittest/mock.py:939: AssertionError
=========================== short test summary info ============================
FAILED tests/test_cli.py::TestCompressSemantics::test_bad_file_format - AssertionError: expected call not found.
Expected: exit("Error loading '/tmp/guix-build-python-tszip-0.2.5.drv-0/tszip_cli_1yijv2dh/msprime.trees': File not in kastore format. Either the file is corrupt or it is not a tskit tree sequence file. It may be a legacy HDF file upgradable with `tskit upgrade` or a compressed tree sequence file that can be decompressed with `tszip`.")
  Actual: exit("Error loading '/tmp/guix-build-python-tszip-0.2.5.drv-0/tszip_cli_1yijv2dh/msprime.trees': File not in kastore format. Either the file is corrupt or it is not a tskit tree sequence file. It may be a legacy HDF file upgradable with `tskit upgrade` from tskit version<0.6.2 or a compressed tree sequence file that can be decompressed with `tszip`.")
=================== 1 failed, 90 passed, 2 skipped in 2.20s ====================
```

--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -287,13 +287,11 @@ class TestCompressSemantics(TestCli):
         with mock.patch("tszip.cli.exit", side_effect=TestException) as mocked_exit:
             with self.assertRaises(TestException):
                 self.run_tszip([str(self.trees_path)])
-            mocked_exit.assert_called_once_with(
-                f"Error loading '{self.trees_path}': File not in kastore format. Either"
-                f" the file is corrupt or it is not a tskit tree sequence file. It may"
-                f" be a legacy HDF file upgradable with `tskit upgrade` from tskit "
-                f"version<0.6.2 or a compressed tree sequence file that can be "
-                f"decompressed with `tszip`."
-            )
+            args = mocked_exit.call_args[0]
+            self.assertEqual(len(args), 1)
+            error_message = args[0]
+            self.assertIn("Error loading", error_message)
+            self.assertIn("File not in kastore format", error_message)
 
     def test_compress_stdout_keep(self):
         self.assertTrue(self.trees_path.exists())

